From 1903bc2e4aee1e12555f24ce9416e294faf99d8c Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Mon, 24 Jan 2022 17:57:34 +0200
Subject: [PATCH] [SW-58071] gaudi2: use pre compiled rom symbols

Since the addition of gaudi3 changes in ROM code base, the code
compiled for gaudi2 ROM doesn't match exactly with ROM code on
device due to which ppboot fails to find exact symbols of ROM
when ran on device. In order to fix this issue, this patch add
pre-compiled ROM symbols elf which will be used for gaudi2 ppboot
compilation for ARC target.

Change-Id: I656632d6ace6285a62cea0fb6d1ed3be7bc0974d
---

diff --git a/board/habanalabs/gaudi2_arc/exports.elf b/board/habanalabs/gaudi2_arc/exports.elf
new file mode 100755
index 0000000..cd3abf1
--- /dev/null
+++ b/board/habanalabs/gaudi2_arc/exports.elf
Binary files differ
